### DISCUSSION TITLE
Blackhole, Rekordbox, OBS and Pioneer DJ controller

### DISCUSSION THREAD

**Discussion by dannyw76** [ExistentialAudio/BlackHole#270] [Q&A]
Hi Everyone,

Very new to using Blackhole and OBS so please excuse if this sounds like the kind of question a simpleton would ask!

When using Blackhole and OBS the output seems like its coming from the individual channels rather than the master. What I mean is, adjusting the master volume on my Pioneer DJ controller makes no impact on the volume of OBS and Twitch. If I alter the channel volume I can see the changes. Is this to be expected, I would have thought the changes should be coming through from the Master (In my head I'm setting this up like a club, so the master volume is in control rather than each channel).

Any help and advice would be greatly appreciated!

Thanks,

Danny

**Comment by devinroth** [comment]
Are you using a multi-output or an aggregate?
